MAUI/DataForm/Validation.md

```diff
@@ -0,0 +1,276 @@
+
+{% tabs %}
+{% highlight C# %}
+
+    [Required(AllowEmptyStrings = false, ErrorMessage = "Name should not be empty")]
+    [StringLength(15, ErrorMessage = "Name should not exceed 15 characters")]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+#### Date range attribute
+
+You can validate the date time value using date range attribute.
+
+{% tabs %}
+{% highlight C# %}
+
+    [DataType(DataType.Date)]
+    [DataFormDateRange(MinimumDate = "01/01/2022", MaximumDate = "31/12/2022", ErrorMessage = "Join date is invalid")]
+    public DateTime JoinDate { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Validation mode
+
+The [ValidationMode] determines when the value should be validated.
+
+The supported validation modes are as follows:
+
+* LostFocus
+* PropertyChanged
+* Manual
+
+{% tabs %}
+{% highlight XAML %}
+<ContentPage 
+...
+             xmlns:dataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm">
+    <dataForm:SfDataForm
+        x:Name="dataForm" 
+        ValidationMode="LostFocus">
+    </dataForm:SfDataForm>
+
+</ContentPage>
+
+{% endhighlight %}
+{% highlight C# %}
+
+this.dataForm.ValidationMode = DataFormValidationMode.LostFocus;
+
+{% endhighlight %}
+{% endtabs %}
+
+#### LostFocus
+
+If the validation mode is `LostFocus`, the value will be validated when the editor lost its focus. By default the [ValidationMode] is `LostFocus`
+
+#### PropertyChanged
+
+If the validation mode is `PropertyChanged`, the value will be validated immediately when it is changed.
+
+#### Manual
+
+If the validation mode is `Manual`, the value should be validated manually by calling the [SfDataForm.Validate] or [SfDataForm.Validate(new List<string>())] method.
+
+The following code validates the value of all the properties in the data object:
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.Validate();
+
+{% endhighlight %}
+{% endtabs %}
+
+To validate the specific list of property value, pass the property names as argument.
+
+{% tabs %}
+{% highlight C# %}
+
+this.dataForm.Validate(new List<string>() {"FirstName", "Age" });
+
+{% endhighlight %}
+{% endtabs %}
+
+You can determine whether the data form or property is valid or not by using the `Validate` method.
+
+{% tabs %}
+{% highlight C# %}
+
+        bool isValid = this.dataForm.Validate();
+
+        List<string> strings = new List<string>();
+        strings.Add("FirstName");
+        bool isPropertyValid = this.dataForm.Validate(strings);
+
+{% endhighlight %}
+{% endtabs %}
+
+If the data form or property is valid, `true` will be returned. Or else `false` will be returned.
+
+## Valid message
+
+If the values are correct, you can show the [ValidMessage]. As like error message, the valid message will also be displayed at the bottom of the editor.
```

Comment:
If the values are correct, you can show the [ValidMessage] as like error message,

Reply:
content changed